src/pages/index.js

```diff
@@ -36,7 +62,15 @@ const cardSection = new Section(
   ".cards__list"
 );
 
-cardSection.renderItems();
+api
+  .getAppInfo()
+  .then(([userData, cardsData]) => {
+    userInfo.setUserInfo({ name: userData.name, job: userData.about });
+    cardSection.renderItems(cardsData);
+  })
```

Comment:
Please, delete `api.getAppInfo()` because you've already fetched the profile data and the cards. Or delete those requests from the top of the file not to duplicate fetching

Reply:
Another remark hasn't been corrected https://github.com/chriskapano/se_project_aroundtheus/pull/1#discussion_r1810099899 


You have everything in lines 30 and 41 separately. Or you need to delete those requests and leave this one